CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 CORTX is about building the world's best scalable mass-capacity object storage system. If you’re interested in what we’re building and intrigued by hard challenges, here's everything you need to know about contributing to the project and how to get started. 
 
-This guide is intended to provide quick start instructions for developers who want to build, test, and contribute to the CORTX software.  If you are merely looking to _test_ CORTX, please use [these instructions](doc/ova/1.0.4/CORTX_on_Open_Virtual_Appliance.rst).
+This guide is intended to provide quick start instructions for developers who want to build, test, and contribute to the CORTX software.  If you are merely looking to _test_ CORTX, please use [these instructions](doc/ova/2.0.0/PI-3/CORTX_on_Open_Virtual_Appliance_PI-3.rst).

```

Comment:
@hseow-seagate, you can update the latest OVA link as published in https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-4/CORTX_on_Open_Virtual_Appliance_PI-4.rst

Reply:
Done

Reply:
updated link is broken

Reply:
Hi @mukul-seagate11, the updated link is not broken. I did not update branch that's why its broken when trying to access the link from patch-12
I have now updated branch to patch-12. The link works fine now